doc/nrf/gs_modifying.rst

```diff
@@ -151,8 +221,39 @@ To locate a specific configuration option, use the filter (:guilabel:`Jump to` i
 The documentation for each :ref:`configuration option <configuration_options>` also lists the menu path where the option can be found.
 
 .. important::
-   All changes to the :file:`.config` file are lost when you clean your build directory.
-   You can save it to another location, but you must then manually copy it back to your build directory.
+   All changes to the :file:`.config` file are lost when you clean your build directory or you rebuild the project with the ``pristine`` target.
+   You can save the file to another location, but you must then manually copy it back to your build directory.
+   See Zephyr's :ref:`zephyr:application_rebuild` for more information.
```

Comment:
Cut this line as well, as it doesn't make sense without the two lines above.

Reply:
Moved next to "This means that this file is available when building the application, but it is deleted when you clean the build directory." as discussed on Teams.